### PR TITLE
Task 8 patch review

### PR DESCRIPTION
**Pull request by Alexander-Timmins**

### CODE REVIEW

File: controller.js [hunk 1]
```diff
@@ -44,11 +45,22 @@ exports.getReviewComments = (req, res, next) => {
     .catch(next);
 };
 
+exports.updateReviewVotes = (req, res, next) => {
+  const reviewId = req.params.review_Id;
+  const vote = req.body.inc_votes;
+  returnUpdatedReview(reviewId, vote)
+    .then((review) => {
+      res.status(202).send({ updatedReview: review });
```
Comment (HeatherEver) [inline controller.js:53]: we would normally just send a general 200 status for successful requests :)

File: __tests__/app.test.js [hunk 2]
```diff
@@ -243,4 +244,61 @@ describe('app.js', () => {
...
+        .expect(404);
+    });
+    test('returns a status 400 when passed an invalid body', () => {
+      const votesToUpdate = {
+        random: 1000,
+      };
+      return request(app)
+        .patch('/api/review/3')
+        .send(votesToUpdate)
+        .expect(400);
```
Comment (HeatherEver) [inline __tests__/app.test.js:301]: Nice work here - there are a couple more tests you could consider, such as sending an empty object and decrementing the votes

**Review by HeatherEver — APPROVED** [review]
Nice work Alex, just a couple of things to have a think about
